src/AzOps.psd1

```diff
@@ -3,7 +3,7 @@
     RootModule        = 'AzOps.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '1.0.3'
+    ModuleVersion     = '1.0.2'
```

Comment:
Does this need to be iterated to `1.0.4` ?

Reply:
my understanding is we are doing it writeback after the release on PowerShell gallery.